MT/VR-MT-0003/VR-MT-0003.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Identifier": "VR-MT-0003",
+  "Type": "Acceptance",
+  "Country": "MT",
+  "Version": "1.0.1",
+  "SchemaVersion": "1.0.0",
+  "Engine": "CERTLOGIC",
+  "EngineVersion": "0.7.5",
+  "CertificateType": "Vaccination",
+  "Description": [
+    {
+      "lang": "en",
+      "desc": "The vaccination schedule must be complete (e.g., 1/1, 2/2,3/3)."
+    }
+  ],
+  "ValidFrom": "2022-01-15T00:00:00Z",
+  "ValidTo": "2030-06-01T00:00:00Z",
+  "AffectedFields": [
+    "v.0",
+    "v.0.dn",
+    "v.0.sd"
+  ],
+  "Logic": {
+    "if": [
+      {
+        "var": "payload.v.0"
+      },
+      {
+        "===": [
```

Comment:
Please take the following into account: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32021D2301 (issuance shall be in effect from February 1st onwards, so verification shall do likewise)
In other words: you should always accept n/n (as you do), but n/1 as well (which you only do for n=1).